neo/Settings.py

```diff
@@ -19,22 +19,32 @@
 
 from neorpc.Client import RPCClient
 from neorpc.Settings import settings as rpc_settings
+import sys
 
 
-# Create am absolute references to the project root folder. Used for
-# specifying the various filenames.
 dir_current = os.path.dirname(os.path.abspath(__file__))
-DIR_PROJECT_ROOT = os.path.abspath(os.path.join(dir_current, ".."))
+
+IS_PACKAGE_INSTALL = False
+ROOT_INSTALL_PATH = None
+
+# This detects if we are running from
+# an 'editable' version ( like ``python neo/bin/prompt.py`` )
+# or from a packaged install version from pip
+if 'site-packages/neo' in dir_current:
+    ROOT_INSTALL_PATH = os.path.abspath(os.path.join(dir_current, ".."))
+    IS_PACKAGE_INSTALL = True
+else:
+    ROOT_INSTALL_PATH = os.getcwd()
```

Comment:
i don't think that works in all circumstances. Let's say I'm in a subdirectory and execute `python ../prompt.py` then the references based on ROOT_INSTALL_PATH will fail. In this case here I think we should use `ROOT_INSTALL_PATH = os.path.abspath(os.path.join(dir_current, ".."))`

Reply:
Thinking about it, I don't think we need this at all? We could just set `ROOT_INSTALL_PATH = os.path.abspath(os.path.join(dir_current, ".."))` no matter if installed as package or not?